benchmarks/data-serving/client/Dockerfile

```diff
@@ -1,24 +1,19 @@
-FROM cloudsuite/java:openjdk8
-
-ENV CASSANDRA_VERSION 3.11.3
+FROM cloudsuite/cassandra:4.0
 
+ENV CASSANDRA_VERSION 4.0-alpha3
```

Comment:
it's better to set this inside the cassandra base image, which makes it available to all the images using that base image.

Reply:
Set in cassandra base image